FreeRTOS/FreeRTOS-Kernel
# Add missing Task Notification IFDEF

Description
-----------

Wrap the task notification check in vTaskGetInfo() in in a  #if ( con…figUSE_TASK_NOTIFICATIONS == 1 )

Test Steps
-----------


Checklist:
----------



Related Issue
-----------

Created due to this [FreeRTOS Forum post](https://forums.freertos.org/t/compiling-without-task-notification-and-stream-buffers/19045/4) 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
